README.md

```diff
@@ -233,4 +233,6 @@ Your app is ready to be deployed! Note that builds for deployment are not made o
     where u.email = @email
     ```
 
-14. Return to the UI and you should be good to go!
+14. Verify that you have followed the instructions in [Keycloak Advanced Setup](./KeycloakAdvancedSetup.md) to ensure you have the proper permissions within the portal.
```

Comment:
nice catch  - how about we make it even clearer so that the dev doesn't accidentally do some of the steps that aren't required. Perhaps something like:
``
14. Assign yourself the `api-participant-member` role by following these steps: [Assign Role to a Particular User](./KeycloakAdvancedSetup.md#assign-role-to-a-particular-user)
```

Reply:
That's a good idea.  I changed it.